dll/win32/browseui/shellbrowser.cpp

```diff
@@ -3676,6 +3675,17 @@ LRESULT CShellBrowser::OnToggleStatusBarVisible(WORD wNotifyCode, WORD wID, HWND
     return 0;
 }
 
+LRESULT CShellBrowser::CreateStatusBar()
+{
+    if (!fStatusBar)
+    {
+        fStatusBar = CreateWindow(STATUSCLASSNAMEW, NULL, WS_CHILD | WS_VISIBLE | WS_CLIPSIBLINGS |
+                                  SBT_NOBORDERS | SBT_TOOLTIPS, 0, 0, 500, 20, m_hWnd, (HMENU)0xa001,
```

Comment:
Do we know why "0xa001" needs to be used there?

Reply:
Per https://github.com/wine-mirror/wine/blob/master/include/shlobj.h it shows:
```
/* undocumented toolbar items from stddlg's*/
#define FCIDM_TB_UPFOLDER       0xA001
#define FCIDM_TB_NEWFOLDER      0xA002
#define FCIDM_TB_SMALLICON      0xA003
#define FCIDM_TB_REPORTVIEW     0xA004
#define FCIDM_TB_DESKTOP        0xA005  /* FIXME */
````

Reply:
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=FCIDM_TB_UPFOLDER